src/podroller.coffee

```diff
@@ -3,12 +3,15 @@ path        = require 'path'
 express     = require 'express'
 fs          = require "fs"
 http        = require "http"
+https       = require "https"
```

Comment:
This probably doesn't apply much to this project, but I've been using the "follow-redirects" module which provides a drop-in replacement for the standard http/https modules but it follows redirects by default.  So if we have to put in redirects for services our projects access, they won't all break and we won't have to make code changes to keep everything working.

Reply:
Ooh, that's pretty useful. Saving that tidbit for later. Thanks.